pkg/cli/aws/aws.go

```diff
@@ -261,6 +261,7 @@ func runOktaAwsCLI(ctx context.Context, b *box.Config, args ...string) error {
 		"OKTA_ORG_DOMAIN="+b.AWS.Okta.OrgDomain,
 		"OKTA_AWS_ACCOUNT_FEDERATION_APP_ID="+b.AWS.Okta.FederationAppID,
 		"OKTA_OIDC_CLIENT_ID="+b.AWS.Okta.OIDCClientID,
+		"OKTA_AWSCLI_SESSION_DURATION="+b.AWS.Okta.SessionDuration,
```

Comment:
suggestion: you'll need to add `SessionDuration` to a box struct definition for this to work:

https://github.com/getoutreach/gobox/blob/21ac66929735a49726c1ccb5e9648c4d474ba1c7/pkg/box/box.go#L50-L59

Reply:
Thank you.